### PR TITLE
Added Hyperlinks to contact fields

### PR DESCRIPTION
**Pull request by PSNAppz**
* Added hyperlink for request_list.html
* Font Size changed to 16px for readability.

### CODE REVIEW

File: mainapp/templates/mainapp/request_list.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
             {% endif %}
           </td>
           <td>{{ req.requestee }}</td>
-          <td>{{ req.requestee_phone }}</td> 
+          <td><a href="tel:{{ req.phone }}">{{ req.requestee_phone }}</a></td> 
           <td>{{ req.summarise | linebreaks }}</td>
```
Comment (asimqt) [inline mainapp/templates/mainapp/request_list.html]: Should this be `<a href="tel:{{ req.requestee_phone }}">{{ req.requestee_phone }}</a>` instead?

Comment (PSNAppz) [reply mainapp/templates/mainapp/request_list.html]: Thanks for pointing out.

Comment (PSNAppz) [reply mainapp/templates/mainapp/request_list.html]: Fixed.

**Comment by jilvin** [comment]
@PSNAppz No relevant change can be seen in the Files changed section. And are you sure Raleway font is imported for use?

Cheers.

**Comment by PSNAppz** [comment]
@jilvin Raleway font was there before.

**Comment by salmanulfarzy** [comment]
This feature is already merged to master.

**Pull request closed without merging**